src/app/pages/start-wizard/start-wizard.page.html

```diff
@@ -1,265 +1,136 @@
-        </ion-row>
-        <ion-row class="full-grid-row">
-          <ion-col class="ion-align-self-center">
-            <h2 [@headerAnimation4]="state">{{'START_WIZARD.COMPETENCE_PAGE.HEADER'|translate}}</h2>
-            <h3 class="ion-text-wrap">{{'START_WIZARD.COMPETENCE_PAGE.LINE_1'|translate}}</h3>
-          </ion-col>
-        </ion-row>
-      </ion-grid>
-    </ion-slide>
-    <ion-slide>
-      <ion-grid class="full-grid legal-grid">
-        <ion-row class="legal-info-row">
-          <ion-col class="ion-padding ion-align-self-bottom">
-            <div class="legal-icon" [ngClass]="{'hide': !showLegalIcon }">&sect;</div>
-          </ion-col>
-        </ion-row>
-        <ion-row class="legal-text-row">
-          <ion-col align-self-top>
-            <h2 [@headerAnimation5]="state">{{'LEGAL_TERMS.HEADER' | translate}}</h2>
-            <h3 class="ion-no-padding ion-text-wrap">
-              <ul>
-                <li [@legalText1]="state">{{'LEGAL_TERMS.LINE_1' | translate}}</li>
-                <li [@legalText2]="state">{{'LEGAL_TERMS.LINE_2' | translate}}</li>
-                <li [@legalText3]="state">{{'LEGAL_TERMS.LINE_3' | translate}}</li>
-                <li [@legalText4]="state">
-                  <span [innerHtml]="'LEGAL_TERMS.LINE_4' | translate"></span>
-                </li>
-                <li [@legalText5]="state">
-                  <span [innerHtml]="'LEGAL_TERMS.LINE_5' | translate"></span>
-                </li>
-                <li [@legalText6]="state">
-                  <span [innerHtml]="'LEGAL_TERMS.LINE_6' | translate:{legalUrl: legalUrl}"></span>
-                </li>
-              </ul>
-            </h3>
-          </ion-col>
-        </ion-row>
-      </ion-grid>
-    </ion-slide>
-  </ion-slides> -->
-</ion-content>
+<swiper-container
+  #swiper
+  slides-per-view="auto"
+  loop="false"
+  css-mode="false"
+  navigation="true"
+  pagination="true"
+  space-between="0"
+  centered-slides="false"
+  center-insufficient-slides="false"
+  zoom="true"
+  keyboard="false"
+  (swiperslidechange)="onSlideChange($event)"
+>
+  <swiper-slide>
+    <div class="slide__container">
+      <div class="svg__container">
+        <svg-icon src="/assets/images/onboarding/eclipse.svg"></svg-icon>
+      </div>
+      <div class="text__container" style="max-width: 230px">
+        <h2>
+          {{ "SETTINGS.LANGUAGE" | translate }}
+          @if (language !== LangKey.en) {
+            <span> / Language</span>
+          }
+        </h2>
+        <ion-list class="language-list">
+          <ion-item class="ion-no-padding">
+            <ion-label position="stacked" color="medium">
+              {{ "START_WIZARD.CHOOSE_LANGUAGE.HEADER" | translate }}
+              @if (language !== LangKey.en) {
+                <span> / Choose language</span>
+              }
+            </ion-label>
+            <ion-select
+              cancelText="{{ 'DIALOGS.CANCEL' | translate }}"
+              [(ngModel)]="language"
```

Comment:
Merk at hvis du endrer language til å være en signal så må du skrive om dette litt. Kan evt hjelpe deg med det. Men bare behold det sånn hvis du ikke gidder endre language.
Dette er jo ikke kode du har laget, men den er ikke veldig god nå, siden det er to-veis binding + bruk av event for å lagre språk.

Reply:
jeg har gjort den om litt. nå brukes language kun som default verdi, men selve overskriving skjer via saveLanguage. grunnen? jeg kan ikke skrive i signaler opprettet med toSignal, men jeg syns det her fungerer bra.